### PR TITLE
OFAST-172 Playground Problem Creator

### PR DESCRIPTION
**Pull request by AsirAAlam**
- added a problem creator section to the playground

https://github.com/ofast-team/frontend/assets/46213673/d1c94266-d0fc-47f5-b8a7-19ab29f807d4

### CODE REVIEW

**Comment by gblake31** [comment]
This is really well done. I think its worth showing this in our demo to prove that the content is easily expandable. Is there anything we could do here to make the page a little more user friendly? (after STEM Day)

When the input fields are filled in, it can be easy to get lost. 
image
Maybe we could put labels next to the fields instead of in them? Also might be nice to have short descriptions for the fields that need it (like "in MB") (or "required") As a contributor, I wouldn't want to constantly refer back to the tutorial.

**Comment by AsirAAlam** [comment]
Added labels and some info

https://github.com/ofast-team/frontend/assets/46213673/e079076f-59fa-4b26-97f8-e49bd2d1d510

File: src/components/PlaygroundPage/ProblemCreatorTab.tsx [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+      return {
+        ...curProblemData,
+        resources: newResources,
+      }
+    })
+  }
+
+  const removeItem = (key: string) => {
+    setProblemData((curProblemData) => {
+      const newEntries = curProblemData[key].slice(0, -1)
```
Comment (sd0623) [inline src/components/PlaygroundPage/ProblemCreatorTab.tsx:80]: nit: what if I want to remove only first tag/resource?

**Review by sd0623 — APPROVED** [review]
Looks good. Just left a comment.

**Comment by AsirAAlam** [comment]
Added delete buttons

image
image

**Pull request merged**